ci/pipelines/ubuntu-noble/stemcells.yml

```
@@ -0,0 +1,25 @@
+  include_fips_iaas: []
+  os: noble
+  os_version: "24.04"
+  os_name: ubuntu-noble
+  subnet_int: "0"
+
+blobstore_types:
+- dav
+- gcs
+- s3
+- azure-storage
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)